styles/base/_colors.scss

```diff
@@ -9,6 +9,9 @@ $blue: #36a;
 $purple: #636;
 $rose: #957;
 $red: #a22;
+$orange: #a36036;
+$yellow: #a1a336;
+$violet: #8F00FF;
```

Comment:
@willuhm-js The violet is too vibrant to my eyes—what do you think about `#838`? (screenshot below)

<img width="1088" alt="Screen Shot 2022-02-05 at 12 42 16 PM" src="https://user-images.githubusercontent.com/872474/152658261-d04d72a8-9558-4d77-81e3-074c1532f6b8.png">